<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20160729 16:37:35.182" generator="Robot 2.9.2 (Python 2.7.12 on linux2)">
<suite source="/home/huangjm/study/robot/sniff/parse.robot" id="s1" name="Parse">
<test id="s1-t1" name="TestParse">
<kw name="Create Session">
<doc>*Creating keyword failed:* No keyword with name 'Create Session' found.</doc>
<arguments>
<arg>sniff_session</arg>
<arg>http://192.168.16.164:10990</arg>
</arguments>
<msg timestamp="20160729 16:37:35.309" level="FAIL">No keyword with name 'Create Session' found.</msg>
<status status="FAIL" endtime="20160729 16:37:35.309" starttime="20160729 16:37:35.308"></status>
</kw>
<status status="FAIL" endtime="20160729 16:37:35.310" critical="yes" starttime="20160729 16:37:35.301">No keyword with name 'Create Session' found.</status>
</test>
<status status="FAIL" endtime="20160729 16:37:35.311" starttime="20160729 16:37:35.187"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" id="s1" name="Parse" pass="0">Parse</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20160729 16:37:35.299" level="ERROR">Error in file '/home/huangjm/study/robot/sniff/parse.robot': Test library '../../../Python27/Lib/site-packages/RequestsLibrary/RequestsKeywords.py' does not exist.</msg>
</errors>
</robot>
